### PR TITLE
Add `keys` and `excludeKeys` options to REST docs

### LINKED ISSUE #632: 📕 Add documentation for query exclude keys
See parse-community/parse-server#5737 for the server implementation.
See parse-community/parse-sdk-js#857 for the JS SDK implementation.
See parse-community/parse-php-sdk#445 for the PHP SDK implementation.

### PR DESCRIPTION
**Pull request by RaschidJFR**
Closes #632

### CODE REVIEW

**Comment by TomWFox** [comment]
Thanks for doing that, I will review this tomorrow.

**Review by TomWFox — CHANGES_REQUESTED** [review]
JS changes look good 👍

I don't think `excludeKeys` should be mentioned in the retrieving objects section in the REST guide. If you look at the [query constraints](http://docs.parseplatform.org/rest/guide/#query-constraints) section there is a part which would make more sense...
> In addition to where, there are several parameters you can use to configure what types of results are returned by the query.

I think `excludeKeys` should be added in the table here after `keys` and an example added after the `keys` example.

**Comment by RaschidJFR** [comment]
You're right  @TomWFox ! I let `include` deceive me. BTW I also added padding to `td` elements as tables started to look a bit tight.

**Review by TomWFox — APPROVED** [review]
Looks good, tables look much better!

**Comment by TomWFox** [comment]
@dplewis could you give the code samples a quick once over?

File: _includes/rest/queries.md [hunk 1]
```diff
@@ -302,13 +302,14 @@ print result
...
-| skip        | Use with limit to paginate through results        |
-| keys        | Restrict the fields returned by the query         |
-| include     | Use on Pointer columns to return the full object  |
+| Parameter     | Use                                               |
+|-------------------------------------------------------------------|
+| order         | Specify a field to sort by                        |
+| limit         | Limit the number of objects returned by the query |
+| skip          | Use with limit to paginate through results        |
+| keys          | Restrict the fields returned by the query         |
+| excludeKeys   | Exlcude specific fields from the returned query   |
```
Comment (dplewis) [inline _includes/rest/queries.md]: Typo *Exclude specific fields...

**Review by dplewis — CHANGES_REQUESTED** [review]
Looks good to me. Just a quick nit

**Pull request merged**